spec/package_spec.rb

```diff
@@ -280,11 +266,19 @@ def build_app_source_with_files(files)
     end
   end
 
-  describe '#market_translations!' do
-    let(:translations) { { 'app' => { 'name' => 'Some App', 'description' => 'It does something.' } } }
+  describe '#runtime_translations' do
+    let(:translations) do
+      {
+        'app' => {
+          'name' => 'Some App',
+          'description' => 'It does something.',
+          'everything_else' => 'preserved'
+        }
+      }
+    end
     let(:source) { build_app_source(additional_files: { "translations/en.json" => translations.to_json }) }
 
-    subject { package.market_translations!('en') }
+    subject { package.send :runtime_translations, package.translations_for('en')['app'] }
```

Comment:
why not? :trollface: 

I think you should always use `fetch`, unless you _expect_ a missing key. It makes failures happen earlier with easier to diagnose messages.